spec/cache/senders/impressions_sender_spec.rb

```diff
@@ -7,6 +7,7 @@
     let(:repository) { SplitIoClient::Cache::Repositories::ImpressionsRepository.new(adapter, config) }
     let(:sender) { described_class.new(repository, config, nil) }
     let(:formatted_impressions) { sender.send(:formatted_impressions, repository.clear) }
+    let(:ip) { Socket.ip_address_list.detect { |intf| intf.ipv4_private? }.ip_address }
```

Comment:
What happens if ip could not be fetched? just like in config_split.rb unknown is set on that case, but I cannot see that here. @maksimf @patricioe